bin/celeste.jl

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/env julia
 
 using DocOpt
+using Dtree
 import Celeste
 
+const cpu_hz = 2.3e9
+
+const wira = 0.015
```

Comment:
These are the width of ra / dec rectangles.  Could you please add a comment to this effect?